cooltools/dotfinder.py

```diff
@@ -1065,22 +1071,56 @@ def scoring_step(clr, expected, expected_name, tiles, kernels,
         # consider using
         # https://github.com/mirnylab/cooler/blob/9e72ee202b0ac6f9d93fd2444d6f94c524962769/cooler/tools.py#L59
         # here:
+        chunks = map_(job, tiles, **map_kwargs)
         ###########################################
         #
         # this is to be rewritten using cooler output
         # as in https://github.com/mirnylab/cooltools/issues/51#issuecomment-458664781
         #
         ###########################################
-        chunks = map_(job, tiles, **map_kwargs)
-        append = False
-        for chunk in chunks:
-            chunk.to_hdf(output_path,
-                         key='results',
-                         format='table',
-                         complevel=9,
-                         complib="blosc:snappy",
-                         append=append)
-            append = True
+        if output_mode == "hdf":
+            print("pandas hdf output ...")
+            append = False
+            for chunk in chunks:
+                chunk.to_hdf(output_path,
+                             key='results',
+                             format='table',
+                             complevel=9,
+                             complib="blosc:snappy",
+                             append=append)
+                append = True
+        # ###########################################
+        # #
+        # # cooler as an output for the dump:
+        # #
+        # ###########################################
+        if output_mode == "cooler":
```

Comment:
does stuff below make sense @nvictus  ?

so far cooler is slower than pytables hdf - don't know how would it scale with # of pixels, if it's more of a constant time, maybe it's ok ...
At the same time cooler comes out twice as small !

I provided some numbers here https://github.com/mirnylab/cooltools/issues/51#issuecomment-459924382